disnake/colour.py

```diff
@@ -128,6 +128,11 @@ def from_hsv(cls: Type[CT], h: float, s: float, v: float) -> CT:
         rgb = colorsys.hsv_to_rgb(h, s, v)
         return cls.from_rgb(*(int(x * 255) for x in rgb))
 
+    @classmethod
+    def from_hex(cls: Type[CT], hex: str) -> CT:
+        """Constructs a :class:`Colour` from hex value."""
```

Comment:
```suggestion
        """Constructs a :class:`Colour` from hex value.
        
        .. versionadded:: 2.5
        
        Parameters
        ----------
        hex: :class:`str`
            The hex to convert to a colour.
            
        Raises
        ------
        ValueError
            The provided value was not able to be converted to an integer.
        """
```

Reply:
If we do this here, we should probably do so for all other `from_x` constructors too.